meta.go

```diff
@@ -28,10 +28,10 @@ var fprintf = fmt.Fprintf
 var metaKeyValidator = regexp.MustCompile(`^\w+(((\[\]|\[(0|[1-9]\d*)\]))?(\.\w+)*)*$`)
 var rightBracketRegExp = regexp.MustCompile(`\[(.*?)\]`)
 
-const metaFile = "meta.json"
+var metaFile = "meta.json"
```

Comment:
I think the default value of `metaFile` should be defined in `main` like `metaSpace`.

Reply:
Updated!